GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiableAccountDetail.h

```diff
@@ -22,6 +26,8 @@ NS_ASSUME_NONNULL_BEGIN
 typedef NS_ENUM(NSInteger, GIDAccountDetailType) {
   /// User account detail for age over 18.
   GIDAccountDetailTypeAgeOver18,
+  /// Unknown or unspecified verification error.
+  GIDAccountDetailTypeUnknown,
```

Comment:
This isn't _quite_ an error case. Update the comment to something like, "Unknown account detail type used when the account detail type is unspecified or cannot be matched."

Also, let's make this the first case.

Reply:
Done.